src/frontend/Elaborator.ml

```diff
@@ -274,6 +274,9 @@ and chk_tm : CS.con -> T.Chk.tac =
     | CS.CofBoundary c ->
       R.Cof.boundary (chk_tm c)
 
+    | CS.CofNeg c ->
+      R.Cof.neg (chk_tm c)
```

Comment:
`@@`

Reply:
Do you want to change all the surrounding code?

Reply:
oh lol let's not worry about it then